src/jquery.uls.core.js

```diff
@@ -61,6 +61,15 @@
 				// Language is unknown to ULS.
 				delete this.languages[ code ];
 			}
+
+			// Check if the language code is a redirect
+			var target = $.uls.data.isRedirect( code );
```

Comment:
`this.options.languages` is initialized to `$.uls.data.getAutonyms()` by default which does not have redirects listed](https://github.com/wikimedia/jquery.uls/blob/8b104212219ed3bd1b203a7c62a68318c637d3cb/src/jquery.uls.data.utils.js#L93-L105). 

I assume somewhere some custom language options are being passed that necessitates this check. Do we know where this is being done?